openpype/hosts/flame/hooks/pre_flame_setup.py

```diff
@@ -33,15 +36,24 @@ def execute(self):
 
         """Hook entry method."""
         project_doc = self.data["project_doc"]
+        project_name = project_doc["name"]
+
+        # get image io
+        project_anatomy = get_anatomy_settings(project_name)
```

Comment:
You can use `Anatomy` object stored in `data`
```suggestion
        project_anatomy = self.data["anatomy"]
```